addon/styles/addon.scss

```diff
@@ -3,9 +3,9 @@
 .frost-object-browser {
   display: flex;
   flex-direction: column;
-  padding: 0 30px;
 
   .frost-info-bar {
+    padding: 0 30px;
```

Comment:
@sglanzer Can you confirm that these styles are desired globally? (`.frost-info-bar` and `.frost-info-bar .title`)